code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -152,5 +152,24 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/item/radio/intercom, 26)
 	set_frequency(1481)
 	set_broadcasting(TRUE)
 
+/obj/item/radio/intercom/command
+	name = "command intercom"
+	desc = "The command team's special extended-frequency intercom. Mostly just used for eavesdropping, gossiping about subordinates, and complaining about the higher-ups."
+	icon_state = "intercom_command"
+	freerange = TRUE
+
+/obj/item/radio/intercom/command/set_frequency(new_frequency)
```

Comment:
This is not the proper way to do it, we should still be calling parent here.

An idea I've got is to make 'syndie' an arg in ``sanitize_frequency`` to make sure non-syndicate headsets can't use it at all, much like 'free' is.